haas/api.py

```diff
@@ -671,7 +703,9 @@ def switch_register_port(switch, port):
     db.session.commit()
 
 
-@rest_call('DELETE', '/switch/<switch>/port/<path:port>')
+@rest_call('DELETE', '/switch/<switch>/port/<path:port>', Schema({
```

Comment:
Yes, it will. Schema doesn't see this; decoding happens in flask or the main web server.